base/dsfs/dataset_test.go

```diff
@@ -334,9 +334,9 @@ func TestDatasetSaveEvents(t *testing.T) {
 	privKey := testPeers.GetTestPeerInfo(10).PrivKey
 	bus := event.NewBus(ctx)
 
-	fired := map[event.Type]int{}
-	bus.Subscribe(func(ctx context.Context, t event.Type, payload interface{}) error {
-		fired[t]++
+	fired := map[event.Topic]int{}
```

Comment:
we switched away from `Topic` to `Type` to match react's (ok, redux) event system, which uses the `type` keyword to demux action types. Let's keep `Type`, and ideally remove any erroneous references to `topic` still hanging around